_includes/pbc/all/install-features/202410.0/install-the-payments-feature.md

```diff
@@ -24,19 +25,237 @@ To start the feature integration, overview and install the necessary features:
 Install the required modules using Composer:
 
 ```bash
-composer require "spryker-feature/payments:{{page.version}}" "spryker/checkout-rest-api:^3.0.0" --update-with-dependencies
+composer require "spryker-feature/payments:{{page.version}}" "spryker/checkout-rest-api:^3.0.0" "spryker/payment-cart-connector"  --update-with-dependencies
 ```
 
+
 {% info_block warningBox "Verification" %}
 Make sure that the following modules have been installed:
 
-| MODULE | EXPECTED DIRECTORY |
-| --- | --- |
-| PaymentDataImport | vendor/spryker/payment-data-import |
-| PaymentGui | vendor/spryker/payment-gui |
+| MODULE                    | EXPECTED DIRECTORY                               |
+|---------------------------|--------------------------------------------------|
+| PaymentDataImport         | vendor/spryker/payment-data-import               |
+| PaymentGui                | vendor/spryker/payment-gui                       |
+| PaymentApp                | vendor/spryker/payment-app                       |
+| PaymentAppExtension       | vendor/spryker/payment-app-extension             |
+| PaymentAppShipment        | vendor/spryker/payment-app-shipment              |
+| PaymentAppWidget          | vendor/spryker-shop/payment-app-widget           |
+| PaymentAppWidgetExtension | vendor/spryker-shop/payment-app-widget-extension |
```

Comment:
should we add that these modules are `required only for ACP Payments`?

Reply:
Yes, we can this column to the table, but they will be a part of the payment feature.